core/src/i18n/en.yaml

```diff
@@ -1617,19 +1617,26 @@ stateful-sets:
   name_singular: Stateful Set
   replicas: Replicas
   title: Stateful Sets
-subscription:
+subscriptions:
   create:
     labels:
+      add-filter: Add filter
```

Comment:
Can you show me where this is displayed? The reason I'm asking is it's a sentence, but doesn't end with a comma.

Reply:
<img width="179" alt="Screen Shot 2022-01-14 at 11 56 21" src="https://user-images.githubusercontent.com/40663133/149504491-01b6a938-e742-4522-bb32-3a5990f05fa8.png">
It's in a button

Reply:
Ah, then all good, thank you :)